data/human/events.txt

```diff
@@ -1167,12 +1167,95 @@ event "navy occupying the south"
 event "catalytic ramscoop available"
 	outfitter "Deep Sky Basics"
 		"Catalytic Ramscoop"
-
-
-
-event "fw gets catalytic ramscoop"
-	outfitter "Delta V Advanced"
-		"Catalytic Ramscoop"
+	fleet "Republic Logistics"
```

Comment:
Why are the logistics fleets changed here instead of in `event "navy using mark ii ships"`?

Reply:
No other Mark II ship uses Catalytic Ramscoops and the Catalytic Ramscoop becomes publicly available separately from the Mark II variant's introduction. If i'm not mistaken, only afterwards is the Catalytic Ramscoop introduced. To avoid the player finding the Catalytic Ramscoop too long before they receive the mission in which you steal the blueprints for the FW through a sympathiser, i choose to take this route. If it isn't actually conflicting in a significant way, i can simply swap them so they are further down, though. Or i could swap it further down anyways, as the navy may have gotten early access to the tech. 

What do you think?

Reply:
The Catalytic Ramscoop is made available before Mark II ships are introduced.

Reply:
Then it is best to keep these separate, so they show up sooner, as that's when they are made available. Thank you for the help!